README.md

```diff
@@ -1 +1,31 @@
-# AIDE boshrelease
+# AIDE bosrelease
+
+A boshrelease for the Advanced Intrusion Detection Environment
+
+## Using it
+
+This release is made with the following assumptions:
+
+- you're using the [node_exporter](https://github.com/bosh-prometheus/node-exporter-boshrelease) bosh release to export stats to prometheus
+- node_exporter is loading /var/vcap/data/node_exporer/config/.&ast;.prom
+
+To use this release, install it, then schedule `run-report.sh` via cron. You probably want to use the [cron boshrelease](https://github.com/cloudfoundry-community/cron-boshrelease)
+for this, but nobody's gonna force you to. [RedHat](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/security_guide/sec-using-aide)
```

Comment:
That page _does_ say at most daily, but doesn't say why.  Seems it'd be good to get the results as soon as possible, no?

Reply:
with Tripwire, we did it hourly. I think that's what we'll do here too